pandas/tests/frame/methods/test_astype.py

```diff
@@ -651,3 +651,8 @@ def test_astype_dt64_to_string(self, frame_or_series, tz_naive_fixture, request)
         # For non-NA values, we should match what we get for non-EA str
         alt = obj.astype(str)
         assert np.all(alt.iloc[1:] == result.iloc[1:])
+
+    def test_astype_bytes(self):
+        # GH#39474
+        result = DataFrame(["foo", "bar", "baz"]).astype(bytes)
+        assert result.dtypes[0] == np.dtype("S3")
```

Comment:
this is not correct, it should be `object`

Reply:
This was True on 1.1.5

Reply:
Could remove the test to avoid testing wrong behavior?

Reply:
hmm yeah this is definitely wrong, we should *never* have a numpy string type in output.

Reply:
We have unfortunately been doing this for many releases ..

Reply:
grr really?

Reply:
ok @phofl let's fix this for 1.3 (the S dtypes) and backport this change.

Reply:
Ok all right. Will open issue as follow up